orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -64,6 +66,23 @@ public OrchestrationClient(@Nonnull final AiCoreDeployment deployment) {
     this.deployment = () -> deployment;
   }
 
+  /**
+   * Generate a completion for the given prompt.
+   *
+   * @param prompt The {@link OrchestrationPrompt} to send to orchestration.
+   * @param config The {@link ModuleConfigs} configuration to use for the completion.
+   * @return the completion output
+   * @throws OrchestrationClientException if the request fails.
+   */
+  @Nonnull
+  public CompletionPostResponse chatCompletion(
+      @Nonnull final OrchestrationPrompt prompt, @Nonnull final ModuleConfigs config)
```

Comment:
If we decide to apply the [convenience API discussed in this PR](https://github.com/SAP/ai-sdk-java/pull/127), this changes to:  
```suggestion
      @Nonnull final OrchestrationPrompt prompt, @Nonnull final OrchestrationConfig config)
```